share/revng/test/configuration/revng-qa/for-import-pe.yml

```diff
@@ -3,19 +3,46 @@
 #
 
 tags:
-  - name: windows
+  - name: vc19-x86-64
     variables:
-      CFLAGS:
-        - /DEBUG:FULL
-        - /Z7
-        - /EHsc
-        - /WX
-        - /nologo
-      LDFLAGS:
-        - /NOLOGO
-        - /WX
+      MSVC_TRIPLE: "x86_64-winsdk-vc19-"
+      MSVC_CFLAGS:
+        - -Z7
+        - -EHsc
+        - -O2
+  - name: vc19-i386
+    variables:
+      MSVC_TRIPLE: "i386-winsdk-vc19-"
+      MSVC_CFLAGS:
+        - -Z7
+        - -EHsc
+        - -O2
+  - name: vc16-i386
+    variables:
+      MSVC_TRIPLE: "i386-winsdk-vc16-"
+      MSVC_CFLAGS:
+        - -Z7
+        - -EHsc
+        - -O2
+  - name: msvc-dynamic-linkage
+    variables:
+      MSVC_CFLAGS:
+        - -MD
+  - name: msvc-static-linkage
+    variables:
+      MSVC_CFLAGS:
+        - -MT
+      MSVC_LDFLAGS:
+        - /opt:ref,icf
 sources:
-  - tags: [windows]
+  - tags: [windows, simple-executable]
+    repeat-for:
+      - [vc19-x86-64, msvc-dynamic-linkage]
+      - [vc19-i386, msvc-dynamic-linkage]
+      - [vc16-i386, msvc-dynamic-linkage]
+      - [vc19-x86-64, msvc-static-linkage]
+      - [vc19-i386, msvc-static-linkage]
+      - [vc16-i386, msvc-static-linkage]
```

Comment:
I'm doing static linkage as the part `mmap`'ed binary. And this is just a way of guarding myself from potential issues - if something is wrong with the way we treat unusual statically-linked windows binaries - i'd rather this test also shown it and not only my stuff.

Reply:
Mmmmmh, I don't understand. Can you detail a scenario that this change prevents/makes easier to detect?

Reply:
Since these end up in the `revng-qa.compiled` pile, they go through the usual pipeline, don't they? So in case something within revng breaks and statically linked windows binaries no longer work, these tests would be the first to fire.

Reply:
So this is not strictly related to ABI stuff? Interesting.
I've never seen a statically linked binary, can you send me one?

Reply:
OK. Static linking is not people actually do on Windows, since the compatibility layer offered by the OS are the system DLLs, not the interface with the kernel as in Linux.

So, if this slows things down too much, or somehow forces us to force linker optimizations in completely unrelated parts of the testing suite (the ABI part), I'd drop it.

It's nice to have, but not a real world use case. I don't want to slow down the test suite or make it more complex in order to have "nice to have" things. Let's keep things simple.

Reply:
These are gone.